win32ss/user/ntuser/keyboard.c

```diff
@@ -1270,6 +1270,9 @@ IntMapVirtualKeyEx(UINT uCode, UINT Type, PKBDTABLES pKbdTbl)
 {
     UINT uRet = 0;
 
+    if (uCode == 0)
+        return uRet;
```

Comment:
Is this correct for all types, i.e. does not depend on `Type` value? Is there an API test for this which we can compare with Windows?

Reply:
Is non-sense to attempt to translate a VK of value zero, tested with API Monitor in windows XP using the application that shows the glitch

Reply:
Ok I'm going to make API test for you in a few days.

Reply:
According to: 
https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes
Values start from 0x01, so does not make any sense to attempt to translate 0x0

Reply:
So according to test results, you need to investigate `IntVkToVsc` function - it should not return `128` scan code for a `0` virtual key.